ci/build-all

```diff
@@ -18,4 +18,9 @@ sudo ldconfig
 mkdir build
 mkdir -p $INSTALL_PREFIX
 ./configure --prefix $INSTALL_PREFIX
-sudo make install
+make install
```

Comment:
I guess you mistakenly removed sudo here (where it is needed), while you wanted to remove it from `make check-functional` below :)

Reply:
Ahh crap yes, replaced now, and will watch out on the Travis CI build service now.